README.md

```diff
@@ -11,7 +11,7 @@ git clone https://github.com/EthanHaque/hifz
 cd hifz
 ```
 
-### 2. Create a Virtual Environment (Recommended)
+### 2. Create a Virtual Environment
```

Comment:
This is good because we mentioned in lecture that we should always use virtual environments.